packages/clay-navigation-bar/src/index.tsx

```diff
@@ -48,9 +48,10 @@ const ClayNavigationBar: React.FunctionComponent<IProps> & {
 }: IProps) => {
 	const [expanded, setExpanded] = React.useState(false);
 
-	const activeElementsCount = children.filter(
+	const activeElementsCount = React.Children.map(
+		children,
 		(child) => child.props.active
-	).length;
+	).filter(Boolean).length;
```

Comment:
@matuzalemsteles I think we also need to update the [props](https://github.com/liferay/clay/blob/7fd169fea7aec47018577ee17ba90931dee7c7df/packages/clay-navigation-bar/src/index.tsx#L21) for `ClayNavigationBar` so that it also accepts one item, because right now it looks like this

```jsx
	children: Array<React.ReactElement<React.ComponentProps<typeof Item>>>;
```

And I'm assuming we want 

```jsx
	children: Array<React.ReactElement<React.ComponentProps<typeof Item>>> | React.ReactElement<React.ComponentProps<typeof Item>>;
```

Since you're on PTO right now, I'm not expecting you to reply, but at least I left a "trace"

Reply:
Yeah, I had to update that in the prop too, thanks.

Reply:
no problem